marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/EnvironmentController.java

```diff
@@ -86,6 +86,7 @@ public SystemInfo getSystemInfo(HttpSession session) {
         info.dataHubVersion = versionInfo.getHubVersion();
         info.marklogicVersion = versionInfo.getMarkLogicVersion();
         info.host = hubCentral.getHost();
+        info.pendoKey = "dummy-key-f7d836d7-2afa-40a6-44e4-f32f58cb35a"; //environment.getProperty("pendo.key");
```

Comment:
Can you remove the commented out line? We don't yet know if the property will come from the environment or not, I don't want this to make someone think we're definitely using the environment object.

Reply:
Removed the comment.